syntax = "proto3";

package suggestion;

service SuggestionService {
    rpc SuggestBooks (BookList) returns (BookList);
}

message BookList {
    repeated Book books = 1;
}

message Book{
    int32 bookId = 1;
    string title = 2;
    string author = 3;
}